### PR TITLE
fix: Reorganize subscription status

### PR DESCRIPTION
**Pull request by ashuntu**
Reorganizes the subscription status page (the last past) according to new designs. Notably, this changes some copy, moves management links to footer text links, and uses generally the same look no matter what type of management.

image

*Note that this page will change significantly in the future to support Landscape configuration statuses. This is just a UI update only.*

---

UDENG-5591

### CODE REVIEW

**Comment by ashuntu** [comment]
This removes some of the copy about being an org managed/store managed/manually managed page. I'm not sure if we still want those elements, but I think maybe we could replace the manage subscription link at the bottom with "This subscription is managed by your organization" when org managed. Or, keep it as is. I don't believe we have new designs for this.

**Comment by ashuntu** [comment]
`LandscapePage` can likely be removed after https://github.com/canonical/ubuntu-pro-for-wsl/pull/1015 is merged.

File: gui/packages/ubuntupro/lib/l10n/app_en.arb [hunk 1]
```diff
@@ -35,18 +35,9 @@
...
-    "@manuallyManaged": {
-        "placeholders": {
-            "proDashboardLink": {
-                "type": "String"
-            }
-        }
-    },
+    "ubuntuProEnabled": "Ubuntu Pro is enabled on this machine.",
+    "ubuntuProEnabledInfo": "All Ubuntu WSL instances have access to Ubuntu Pro security features.",
+    "ubuntuProManage": "Manage Ubuntu Pro subscription",
```
Comment (CarlosNihelton) [inline gui/packages/ubuntupro/lib/l10n/app_en.arb:40]:
```suggestion
    "manageUbuntuPro": "Manage Ubuntu Pro subscription",
```

File: gui/packages/ubuntupro/lib/pages/widgets/page_widgets.dart [hunk 2]
```diff
@@ -170,6 +170,76 @@ class _PageContent extends StatelessWidget {
...
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return Pro4WSLPage(
+      body: Padding(
+        padding: const EdgeInsets.fromLTRB(32.0, 24.0, 32.0, 32.0),
+        child: Column(
+          children: [
+            Expanded(
+              child: Row(
```
Comment (CarlosNihelton) [inline gui/packages/ubuntupro/lib/pages/widgets/page_widgets.dart:197]: Why we need the Row here? Wouldn't a Center widget be more appropriate?

File: gui/packages/ubuntupro/test/pages/subscription_status/subscription_status_page_test.dart [hunk 3]
```diff
@@ -7,15 +7,59 @@ import 'package:ubuntu_service/ubuntu_service.dart';
...
 
 void main() {
   group('subscription info:', () {
     final client = FakeAgentApiClient();
     registerServiceInstance<AgentApiClient>(client);
     final info = SubscriptionInfo();
+
+    group('footer links:', () {
+      final landscape = LandscapeSource()..ensureOrganization();
+      testWidgets('user', (tester) async {
```
Comment (CarlosNihelton) [inline gui/packages/ubuntupro/test/pages/subscription_status/subscription_status_page_test.dart:24]: Can we have a similar test case that finds no footer links for the org case?

**Comment by CarlosNihelton** [comment]
>  I think maybe we could replace the manage subscription link at the bottom with "This subscription is managed by your organization" when org managed.

I like this suggestion better. I'm missing some feedback about this. We might ask Juan about it during our next UX review, if you prefer. I honestly consider that a useful piece of information.

**Review by CarlosNihelton — CHANGES_REQUESTED** [review]
Very clean start. I like the way you've been introducing those changes piecewise. I have few things to address.

File: gui/packages/ubuntupro/test/pages/subscription_status/subscription_status_page_test.dart (hunk 3 above)
Comment (ashuntu) [reply gui/packages/ubuntupro/test/pages/subscription_status/subscription_status_page_test.dart:24]: I have `expect(find.text(lang.ubuntuProManage), findsNothing);` in the other org test cases, I figured that was enough?

Comment (CarlosNihelton) [reply gui/packages/ubuntupro/test/pages/subscription_status/subscription_status_page_test.dart:24]: Yeap, that may suffice indeed. But if we go with your suggestion for the org case it'd be good to have it more specific.

Comment (CarlosNihelton) [reply gui/packages/ubuntupro/test/pages/subscription_status/subscription_status_page_test.dart:24]: Now that we're sure we won't get that copy, let's call the day :)

**Review by CarlosNihelton — APPROVED** [review]
Well done!

**Pull request merged**